### PR TITLE
[#IR-307] As a dev, Add forgot password page and interegrated it with back-end code.

### PR DESCRIPTION
**Pull request by chahat-arora**

### CODE REVIEW

File: src/app/account/identify/identify.component.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  public emailFormControl = new FormControl('', [
+    Validators.required,
+    Validators.email
+  ]);
+  public errorMessage: string;
+  public showError = false;
+
+  constructor(private authService: AuthService) {}
+
+  onSubmit(sendOTP: boolean) {
```
Comment (himanshu-garg-165) [inline src/app/account/identify/identify.component.ts]:
```suggestion
  onSubmit(createOTP: boolean) {
```

File: src/app/account/login/login.component.html [hunk 2]
```diff
@@ -28,6 +28,7 @@ <h2>Welcome to iReflect</h2>
                             Password must be at least 8 characters long.
                             </mat-error>
                         </mat-form-field>
+                        <button mat-button class="forgot-password" [routerLink]="['/identify']">forgot password?</button>
```
Comment (himanshu-garg-165) [inline src/app/account/login/login.component.html]:
```suggestion
                        <button mat-button class="forgot-password" [routerLink]="['/identify']">Forgot password?</button>
```

File: src/app/account/identify/identify.component.ts (hunk 1 above)
Comment (chahat-arora) [reply src/app/account/identify/identify.component.ts]: This boolean is used to decide that we should send an email or not. In any case, OTP is created.

File: src/app/account/login/login.component.html (hunk 2 above)
Comment (chahat-arora) [reply src/app/account/login/login.component.html]: done.

**Pull request merged**